libs/ledger-live-common/src/families/cosmos/preloadedData.mock.ts

```diff
@@ -5,1013 +5,1412 @@ const LEDGER_VALIDATOR_ADDRESS = cryptoFactory("cosmos").ledgerValidator;
 // 1685677 and 1685679
 const data: CosmosPreloadData = {
   validators: [
-    {
-      validatorAddress: "cosmosvaloper1qdxmyqkvt8jsxpn5pp45a38ngs36mn2604cqk9",
-      tokens: 94068472660,
-      votingPower: 94068472660 / (0.7202 * 254624972017526),
-      // Doing the tokens / bondedAmount division manually
-      name: "\u771f\u672c\u806a&IOSG",
-      commission: 0.1,
-      estimatedYearlyRewardsRate: 0,
-    },
     {
       validatorAddress: "cosmosvaloper1qwl879nx9t6kef4supyazayf7vjhennyh568ys",
-      tokens: 5614121688371,
-      votingPower: 5614121688371 / (0.7202 * 254624972017526),
       name: "Certus One",
+      tokens: 2366240079223,
```

Comment:
Do you know if these changes are required for the Playwright tests @sarneijim ?

Reply:
We need apply "change" preloadData to read the correct validators. But if some test is failing regarding this we can "update" it